src/hw4/Student.java

```diff
@@ -0,0 +1,119 @@
+package hw4;
+
+public class Student {
+
+    char studentIdNumber;
+    String name;
+    String surname;
+    int course;
+    double scoreMathematics;
+    double scoreEconomics;
+    double scoreForeignLanguage;
+
+
+    public Student(char studentIdNumber, String name, String surname, int course, double scoreMathematics, double scoreEconomics, double scoreForeignLanguage) {
+
+        this.studentIdNumber = studentIdNumber;
+        this.name = name;
+        this.surname = surname;
+        this.course = course;
+        this.scoreMathematics = scoreMathematics;
+        this.scoreEconomics = scoreEconomics;
+        this.scoreForeignLanguage = scoreForeignLanguage;
+
+    }
+}
```

Comment:
It's needed to create a method for average score calculation:
 void avgMarkAllSubjects() {
    double result = (mathAvgMark + economicAvgMark + foreignLangAvgMark) / 3;
    System.out.println("The average mark for all 3 subjects for " + lastName + " is " + result);
  }

Reply:
Main method:
Student studentA = new Student("studentA", 5.5, 2, 4.5);
    Student studentB = new Student("studentB", 7, 7, 4);
    Student studentC = new Student("studentC", 2, 2, 2);
    studentA.avgMarkAllSubjects();
    studentB.avgMarkAllSubjects();
    studentC.avgMarkAllSubjects();